docs/winrt-apis-for-desktop.md

```diff
@@ -0,0 +1,17 @@
+# WinRT API changes for Desktop apps
+
+In WinUI 3 Preview 4, we have made changes to the way Windowing works in Desktop apps, by no longer creating a CoreWindow or CoreApplicationView. As a result, all WinRT APIs dependent on CoreWindow will no longer work in Desktop apps. This document will list out APIs that have been affected by this change, with replacement or workaround APIs that will allow you to achieve the same functionality. 
+
+This doc will continue to be updated as our community files bugs and as we identify more workarounds/replacements. If you are encountering issues with an API not listed here, please go ahead and [file a bug](https://github.com/microsoft/microsoft-ui-xaml/issues/new?assignees=&labels=&template=bug_report.md&title=) on this repo with the API in question and what you are trying to achieve by using it. 
+
+| WinRT API                                                                  | Replacement or Workaround                 | Notes                                             |
+|----------------------------------------------------------------------------|-------------------------------------------|---------------------------------------------------|
+| [`ApplicationViewTitleBar.ExtendViewIntoTitleBar`](https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.core.coreapplicationviewtitlebar.extendviewintotitlebar?view=winrt-19041)                            | `Window.ExtendsContentIntoTitleBar`        |  API reference documentation coming soon. This is a permanent replacement for WinUI 3.                                                |
+| [`CoreWindow.GetForCurrentThread().GetKeyState()`]((https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.corewindow.getkeystate?view=winrt-19041) )                             | [`KeyboardInput.GetKeyStateForCurrentThread`](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.input.keyboardinput.getkeystateforcurrentthread?view=winui-3.0-preview) |                                                   |
```

Comment:
The GetForCurrentThread part isn't important, it's just that CoreWindow isn't supported.